documentation/crl_refresh.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "Puppet Server: Automatic CRL refresh"
+canonical: "/puppetserver/latest/crl_refresh.html"
+---
+
+Starting in version 2.8.0, Puppet Server can automatically reload an updated CRL into the running SSL context, so that the revocation of an agent's certificate no longer requires a restart of the service to take effect. Prior versions required an explicit restart or reload of this service to reload the CRL, resulting in some small amount of downtime to effect the revocation of a certificate. With automatic CRL refresh enabled, revocation is now transparent and requires no service downtime.
```

Comment:
s/effect/affect

Reply:
I'm reading this as "a small amount of downtime to make the revocation happen", in which case "effect" is correct.

Reply:
Sorry, didn't realize there were two on this line, I'm referring to the second–

"...some small amount of downtime to effect the revocation of..." should be "...some small amount of downtime to **affect** the revocation of"

Reply:
Affect is a verb, effect is a noun. In this case, it's affect.

Reply:
Effect is also a verb, see the second definition here: http://www.dictionary.com/browse/effect?s=t That is the usage in the sentence about downtime, if I'm understanding Moses' description here.

Reply:
I agree with @Magisus that effect is technically being used correctly here, but it might be simpler and less confusing to simply say "downtime to revoke a certificate".

Reply:
Apologies - I explicitly intended `effect` the verb form, but I agree it could be clearer. Will update.